service/ziti-tunnel/service/ipc.go

```diff
@@ -127,6 +127,11 @@ func SubMain(ops chan string, changes chan<- svc.Status, winEvents <-chan Window
 						}
 					}
 				}
+				rts.BroadcastEvent(dto.TunnelStatusEvent{
```

Comment:
just a comment, this sends the total status to the UI - i guess i don't understand why? the UI should just get a "the service has just woken up" event and then the UI could come back to get status as needed? I would think we'd send a different event type? i just don't see the need for sending the full state to the UI at this point

Reply:
It looked like timers were stopping and the state was out of whack after a sleep/start. but newest tests on the the UI branch look like the timers are still going for some reason. If someone can validate that the state does not get out of whack, then I do not need the data. But from initial tests, the state gets lost on sleep/wake.

Reply:
Suren also mentioned in his document that this issue is intermittent.